mappings/net/minecraft/server/network/ServerNetworkListenerThread.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/class_2894 net/minecraft/server/network/ServerNetworkListenerThread
+	FIELD field_13033 handlers Ljava/util/List;
+	FIELD field_13034 addresses Ljava/util/HashMap;
+	FIELD field_13036 socket Ljava/net/ServerSocket;
+	FIELD field_13037 networkHandlerManager Lnet/minecraft/class_2577;
+	FIELD field_13038 address Ljava/net/InetAddress;
+	METHOD <init> (Lnet/minecraft/class_2577;Ljava/net/InetAddress;I)V
+		ARG 1 networkHandlerManager
+		ARG 2 address
+	METHOD method_11868 tick ()V
+	METHOD method_11869 removeAddress (Ljava/net/InetAddress;)V
+		ARG 1 address
+	METHOD method_11870 addHandler (Lnet/minecraft/class_2899;)V
+		ARG 1 pendingConnection
```

Comment:
what's with `pendingConnection`?
isnt that the old mojang name for `ServerLoginNetworkHandler`